source/components/handling-events.md

```diff
@@ -0,0 +1,130 @@
+Instead of having to register event listeners on elements you'd like to
+respond to, simply implement the name of the event you want to respond to
+as a method on your component.
+
+For example, imagine we have a template like this:
+
+```hbs
+{{#x-clickable}}
+  This is a clickable area!
+{{/x-clickable}}
+```
+
+Let's implement `x-clickable` such that when it is
+clicked, an alert is displayed:
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    alert("XClickableComponent was clicked!");
+  }
+});
+```
+
+Browser events may bubble up from the target component to each parent component
```

Comment:
this one is tricky, because they bubble up the DOM, not up component hierarchy. this matters for example for block components.